CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v2.11.0-rc.0 / 2019-07-03
+
+* [CHANGE] Remove max_retries from queue_config. #5649
+* [FEATURE] Add $externalLabels for alert unit tests. #5608
+* [FEATURE] Allows globs for rules when unit testing. #5595
+* [FEATURE] Add option to use Alertmanager API v2. #5482
+* [FEATURE] Added humanizePercentage formatting to templates. #5670
+* [FEATURE] Include InitContainers in Kubernetes Service Discovery. #5598
+* [ENHANCEMENT] Reduce allocations in in PromQL aggregations. #5641
+* [ENHANCEMENT] Add storage warnings to LabelValues and LabelNames API results. #5673
+* [ENHANCEMENT] Add 'code' label to HTTP metrics. #5640
+* [ENHANCEMENT] Enable openbsd/arm build. #5696
+* [ENHANCEMENT] Sanitize HTML output. #5665
+* [ENHANCEMENT] Remote-write allocation improvements. #5614
+* [ENHANCEMENT] Query performance improvement: Efficient iteration and search in HashForLabels and HashWithoutLabels. #5707
+* [ENHANCEMENT] Prometheus Monitoring Mixin for Prometheus itself. #4474
+* [ENHANCEMENT] Allow injection of arbitrary headers in promtool. #4389
```

Comment:
The PR title indicates this is intended for auth, and putting secrets on the commandline is not good practice.

Reply:
Do you want me to change anything here? (I agree with your statement, but this is still what was added factually no?)